pkgs/http/README.md

```diff
@@ -87,7 +87,7 @@ import 'package:http/retry.dart';
 Future<void> main() async {
   final client = RetryClient(http.Client());
   try {
-    print(await client.read(Uri.parse('http://example.org')));
+    print(await client.read(Uri.http('example.org', '')));
```

Comment:
I really dislike reading this when the path is empty - as a reader, how do I figure out what that argument is meant to be?